packages/next/src/cli/next-dev.ts

```diff
@@ -108,8 +111,15 @@ const handleSessionStop = async (signal: string | null) => {
   process.exit(0)
 }
 
-process.on('SIGINT', () => handleSessionStop('SIGINT'))
-process.on('SIGTERM', () => handleSessionStop('SIGTERM'))
+process.on('SIGINT', (code) => handleSessionStop(code))
+process.on('SIGTERM', (code) => handleSessionStop(code))
```

Comment:
just sending `handleSessionStop` works as well but typescript seems to do better with this (e.g. if I set `signal: object` this way would give a typescript error, but `process.on("SIGINT", handleSessionStop)` wouldn't raise any errors.